quixstreams/dataframe/dataframe.py

```diff
@@ -966,7 +966,11 @@ def hopping_window(
             name=name,
         )
 
-    def drop(self, columns: Union[str, List[str]]) -> Self:
+    def drop(
+        self,
+        columns: Union[str, List[str]],
+        errors: Literal["ignore", "raise"] = "raise",
```

Comment:
That looks like a boolean. Do you expect there might ever be more than 2 options? Maybe for simplicity it is worth exposing literally `ignore_missing: bool = False`?

Reply:
Hey @gwaramadze , the API is made to match [the same method in Pandas](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.drop.html)

>          
    errors:  {‘ignore’, ‘raise’}, default ‘raise’
        If ‘ignore’, suppress error and only existing labels are dropped.